apps/src/code-studio/pd/foorm/editor/form/FormEditorManager.jsx

```diff
@@ -38,19 +38,18 @@ class FormEditorManager extends React.Component {
     categories: PropTypes.array,
 
     // populated by redux
-    hasError: PropTypes.bool,
-    formQuestions: PropTypes.object,
+    hasJSONError: PropTypes.bool,
     availableForms: PropTypes.array,
+    questions: PropTypes.object,
+    formName: PropTypes.string,
+    formVersion: PropTypes.number,
+    isFormPublished: PropTypes.bool,
     resetAvailableForms: PropTypes.func,
     setLastSaved: PropTypes.func,
     setSaveError: PropTypes.func,
     setFormData: PropTypes.func,
-    setHasError: PropTypes.func,
-    setLastSavedQuestions: PropTypes.func,
-    questions: PropTypes.object,
-    name: PropTypes.string,
-    version: PropTypes.number,
-    isPublished: PropTypes.bool
+    setHasJSONError: PropTypes.func,
+    setLastSavedFormQuestions: PropTypes.func
   };
 
   constructor(props) {
```

Comment:
Rename `resetAvailableOptions` and move action out of constructor (to editor.js).